cpp/include/cuvs/neighbors/ivf_pq_helpers.hpp

```diff
@@ -39,65 +39,337 @@ namespace cuvs::neighbors::ivf_pq::helpers {
 
 namespace codepacker {
```

Comment:
Can we keep the namespace separation but go ahead and add these definitions to `ivf_pq.hpp` please? Now that we're no longer header-only, there is near zero cost to just putting all the API elements for ivf-pq together.